Tests/Site/KeyTest.php

```diff
@@ -27,4 +27,10 @@ public function testGenerateEncode(): void
         $key = Key::generateEncode(16);
         $this->assertEquals(32, mb_strlen($key, '8bit'));
     }
+
+    public function testGenerateRsa()
+    {
+    	$rsa = Key::generateRSAKey();
+    	$this->assertNotEmpty($rsa);
```

Comment:
To make private/public key generating test meaningful, it should assert the public/private key is valid or verified.

Reply:
It's not easy to validate key , am sure phpUnit has now such function to do that

Reply:
If possible, it's fine to do this tests on this PR.

The meaningful tests can be another PR.